### PR TITLE
Print ports in deployment summary.

### PR DESCRIPTION
**Pull request by joonlim**

### CODE REVIEW

**Comment by joonlim** [comment]
@brandonnelson3

File: gke-deploy/core/resource/resource.go [hunk 1]
```diff
@@ -456,6 +456,25 @@ func deploySummaryExtraInfo(obj *Object) (string, error) {
...
 
 func serviceIPs(obj *Object) (string, error) {
+	ports, ok, err := unstructured.NestedSlice(obj.Object, "spec", "ports")
+	if err != nil {
+		return "", fmt.Errorf("failed to get spec.ports field: %v", err)
+	}
+	if !ok || len(ports) == 0 {
+		return "", nil
+	}
+	portMap := ports[0].(map[string]interface{})
```
Comment (brandonnelson3) [inline gke-deploy/core/resource/resource.go]: Is this the right behavior if len(ports) == 2?

File: gke-deploy/core/resource/resource.go [hunk 2]
```diff
@@ -456,6 +456,25 @@ func deploySummaryExtraInfo(obj *Object) (string, error) {
...
+	if !ok {
+		return "", fmt.Errorf("failed to convert port to map")
+	}
+	port, ok, err := unstructured.NestedInt64(portMap, "port")
+	if err != nil {
+		return "", fmt.Errorf("failed to get port field: %v", err)
+	}
+	if !ok {
+		return "", fmt.Errorf("port field is missing")
```
Comment (brandonnelson3) [inline gke-deploy/core/resource/resource.go]: I would guess this actually means the port field is unparsable as an int64, iIwould also guess that missing would manifest as a default int64. Just make sure this error message is correct.

**Comment by joonlim** [comment]
@ImJasonH Could you please merge this?

**Comment by joonlim** [comment]
@leeonlee Could you please merge this?

**Comment by joonlim** [comment]
@ImJasonH @leeonlee ping

**Pull request merged**